src/components/CompareResults/CompareResultsTable.tsx

```diff
@@ -25,8 +24,8 @@ const tableHeaders: CompareResultsTableHeaders[] = [
 
 function CompareResultsTable(props: CompareResultsProps) {
   const { mode } = props;
-  const compareResults: CompareResultsItem[] = useSelector(
-    (state: RootState) => state.compareResults,
```

Comment:
specifying the type is not needed when using typed hooks: https://redux-toolkit.js.org/tutorials/typescript#define-typed-hooks